pennylane/transforms/qcut.py

```diff
@@ -843,3 +846,331 @@ def qcut_processing_fn(
     )
     return result
+
+
+@dataclass()
+class CutStrategy:
+    """
+    A circuit-cutting distribution policy for executing (large) circuits on available (comparably
+    smaller) devices.
+
+    Args:
+        devices (Union[qml.Device, Sequence[qml.Device]]): Single, or Sequence of, device(s).
+            Optional only when ``max_free_wires`` is provided.
+        max_free_wires (int): Number of wires for the largest available device. Optional only when
+            ``devices`` is provided where it defaults to the maximum number of wires among
+            ``devices``.
+        min_free_wires (int): Number of wires for the smallest available device, or, equivalently,
+            the smallest max fragment-wire-size that the partitioning is allowed to explore.
+            When provided, this parameter will be used to derive an upper-bound to the range of
+            explored number of fragments.  Optional, defaults to ``max_free_wires``.
+        num_fragments_probed (Union[int, Sequence[int]]): Single, or 2-Sequence of, number(s)
+            specifying the potential (range of) number of fragments for the partitioner to attempt.
+            Optional, defaults to probing all valid strategies derivable from the circuit and
+            devices.
+        max_free_gates (int): Maximum allowed circuit depth for the deepest available device.
+            Optional, defaults to unlimited depth.
+        min_free_gates (int): Maximum allowed circuit depth for the shallowest available device.
+            Optional, defaults to ``max_free_gates``.
+        imbalance_tolerance (float): The global maximum allowed imbalance for all partition trials.
+            Optional, defaults to unlimited imbalance. Used only if there's a known hard balacing
+            constraint on the partitioning problem.
+
+    **Example**
+
+    .. code-block:: python
+
+        import pennylane as qml
+
+        dev_a = qml.device('default.qubit', wires=4)
+        dev_b = qml.device('default.qubit', wires=6)
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.4, wires=0)
+            qml.RY(0.9, wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.expval(qml.PauliZ(1))
+
+        tape_dag = qml.transforms.tape_to_graph(tape)
+
+    The following cut strategy specifies that a circuit should be cut into between
+    ``2`` to ``5`` fragments, with each fragment having at most ``6`` wires and
+    at least ``4`` wires:
+
+    >>> cut_strategy = qml.transforms.CutStrategy(
+    ...     max_free_wires=6,
+    ...     min_free_wires=4,
+    ...     num_fragments_probed=(2, 5),
+    ... )
+
+    """
+
+    # pylint: disable=too-many-arguments, too-many-instance-attributes
+
+    devices: InitVar[Sequence[qml.Device]] = None
+    max_free_wires: int = None
+    min_free_wires: int = None
+    num_fragments_probed: Union[int, Sequence[int]] = None
+    max_free_gates: int = None
+    min_free_gates: int = None
+    imbalance_tolerance: float = None
```

Comment:
Since the docs shows these attributes without docstrings:
![image](https://user-images.githubusercontent.com/49409390/155596356-9e22bc3f-6ce1-48e8-9610-1d3c600fb9bd.png)

Would it be possible to add quick one line explanations? E.g.,
```
    imbalance_tolerance: float = None
    #: The global maximum allowed imbalance for all partition trials
```
(I'm not completely sure on the UI to get this to show up in the docs so might need some trial and error)

Reply:
Done, the comment should be above the line.
![image](https://user-images.githubusercontent.com/48225584/155619807-e3c73e8c-7834-49ec-bd7f-60ca5ccffdf7.png)

Reply:
Awesome, thanks for figuring out the right orientation for the comment!